src/libs/actions/BankAccounts.ts

```diff
@@ -97,14 +99,20 @@ function getVBBADataForOnyx() {
 
 /**
  * Submit Bank Account step with Plaid data so php can perform some checks.
- *
- * @param {Number} bankAccountID
- * @param {Object} selectedPlaidBankAccount
  */
-function connectBankAccountWithPlaid(bankAccountID, selectedPlaidBankAccount) {
+function connectBankAccountWithPlaid(bankAccountID: number, selectedPlaidBankAccount: PlaidBankAccount) {
     const commandName = 'ConnectBankAccountWithPlaid';
 
-    const parameters = {
+    type ConnectBankAccountWithPlaidParams = {
```

Comment:
Why is this type defined inline while other types exist in separate files or at the top of this file? My understanding was that all types should exist in their own file.